ADVANCED.md

```diff
@@ -677,8 +677,9 @@ import { MultiCheckboxFacet } from "@elastic/react-search-ui-views";
 | field      | String          | yes       |                                                                                |                                                                                                                                                               | Field name corresponding to this filter. This requires that the corresponding field has been configured in `facets` on the top level Provider.                                                                                                             |
 | filterType | String          | no        | "all"                                                                          | "all", "any", "none"                                                                                                                                          | The type of filter to apply with the selected values. I.e., should "all" of the values match, or just "any" of the values, or "none" of the values. Note: See the example above which describes using "disjunctive" facets in conjunction with filterType. |
 | label      | String          | yes       |                                                                                |                                                                                                                                                               | A static label to show in the facet filter.                                                                                                                                                                                                                |
-| show       | Number          | no        | 10                                                                             |                                                                                                                                                               | The number of facet filter options to show before concatenating with a "more" link.                                                                                                                                                                        |
+| show       | Number          | no        | 5                                                                             |                                                                                                                                                               | The number of facet filter options to show before concatenating with a "more" link.                                                                                                                                                                        |
```

Comment:
Because I thought the readme value is incorrect. By default, the `<Facet />` component shows only the first 5 items, not 10.

https://github.com/elastic/search-ui/blob/master/packages/react-search-ui/src/containers/Facet.js#L37

![image](https://user-images.githubusercontent.com/12813750/58643924-e32a4800-831d-11e9-948b-436a40892037.png)

Reply:
@m-sureshraj Good catch, thanks.